### PR TITLE
Don't configure logging in template pytest fixture

### PR DESCRIPTION
**Pull request by deepyaman**
## Development notes

Change should be replicated across:

- [ ] https://github.com/quantumblacklabs/kedro-starter-pyspark
- [ ] https://github.com/quantumblacklabs/kedro-starter-pyspark-iris
- [ ] https://github.com/quantumblacklabs/kedro-starter-spaceflights
- [ ] https://github.com/quantumblacklabs/kedro-starter-pandas-iris

## Checklist

- [x] Read the [contributing](https://github.com/quantumblacklabs/kedro/blob/master/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change and added my name to the list of supporting contributions in the [`RELEASE.md`](https://github.com/quantumblacklabs/kedro/blob/master/RELEASE.md) file
- [ ] Added tests to cover my changes

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: kedro/templates/project/{{ cookiecutter.repo_name }}/src/tests/test_run.py [hunk 1]
```diff
@@ -36,6 +36,7 @@
 To run the tests, run ``kedro test``.
 """
 from pathlib import Path
+from unittest.mock import patch
```
Comment (lorenabalan) [inline kedro/templates/project/{{ cookiecutter.repo_name }}/src/tests/test_run.py:39]: Can we use `mocker.patch` instead? 🙂

**Comment by lorenabalan** [comment]
This type of change will sadly have to be replicated across all relevant starter repos. 🙃 Could you link them here in the PR description once they're up so we have a place where we keep track?

File: kedro/templates/project/{{ cookiecutter.repo_name }}/src/tests/test_run.py (hunk 1 above)
Comment (deepyaman) [reply kedro/templates/project/{{ cookiecutter.repo_name }}/src/tests/test_run.py]: Done! Is the main benefit to not require the additional import/context manager?

Comment (lorenabalan) [reply kedro/templates/project/{{ cookiecutter.repo_name }}/src/tests/test_run.py]: Fewer imports but also consistency with only using one library for testing. `pytest` has pretty much all the functionality that you can find in `unittest` so no point in mixing.

**Review by lorenabalan — APPROVED** [review]

**Comment by lorenabalan** [comment]
Thank you! @deepyaman I believe if you raise the related PRs you'll be set for Hacktoberfest. 😂

**Review by MerelTheisenQB — APPROVED** [review]
👍

**Comment by deepyaman** [comment]
Above are all done. Free t-shirt, here I come!

**Pull request merged**